fuel-vm/src/tests/blockchain.rs

```diff
@@ -2047,3 +2047,180 @@ fn coinbase_works() {
 
     assert_eq!(data.as_ref().unwrap(), &*expected);
 }
+
+#[test]
+fn various_ldc_issues_poc() {
```

Comment:
Is there any way to break this down in to more granular && specific tests? Tests like this are hard to maintain as it's not clear what behavior they are defining.